src/adtfhir_to_opal/adtfhir_to_opal.py

```diff
@@ -13,13 +13,14 @@
 
 
 class Settings(BaseSettings):
-    output_folder: str = "~/opal-output"
```

Comment:
ups maybe this was a mistake

Reply:
warum war der Pfad hier so gesetzt?

Reply:
gute frage, an sich überschreiben wir es aber sowieso wenn wir es in compose ausführen https://github.com/bzkf/diz-in-a-box/blob/master/docker-compose/compose.adtfhir-to-opal.yaml#L13.

Reply:
Ja genau. Ok war nur verwirrt und hab im Kopf nicht mehr zusammen bekommen wo der Pfad her kam.